packages/manager/cypress/e2e/core/cloudpulse/cloudpulse-resource-limit.spec.ts

```diff
@@ -0,0 +1,290 @@
+/**
+ * This test suite validates the behavior of the DBaaS (Database as a Service) Dashboard,
+ * specifically focusing on how the system handles the maximum resource selection limit
+ * for database clusters. It contains two main test cases:
+ * 
+ * Ticket: DI-21897
+ * 
+ * 1. **Max Resource Limit Reached**:
+ *    - Verifies that when the maximum number of database clusters (1) is selected,
+ *      the "Select All" option is hidden and only the available clusters (up to the limit)
+ *      are shown in the `Database Clusters` dropdown.
```

Comment:
available clusters? clusters exceeding the selection limit will be disabled, but they will still be visible in dropdown..